tracer/test/Datadog.Trace.Tools.Runner.IntegrationTests/Checks/IisCheckTests.cs

```diff
@@ -124,7 +124,7 @@ public async Task ListApplications()
 
             using var console = ConsoleHelper.Redirect();
 
-            var result = await CheckIisCommand.ExecuteAsync(new CheckIisSettings { SiteName = "sample/dummy" }, _iisFixture.IisExpress.ConfigFile, _iisFixture.IisExpress.Process.Id);
+            var result = await CheckIisCommand.ExecuteAsync(new CheckIisSettings { SiteName = new[] { "sample/dummy" } }, _iisFixture.IisExpress.ConfigFile, _iisFixture.IisExpress.Process.Id);
```

Comment:
Worth adding a check for the site name joining, e.g. `new[] { "sample", "dummy" }` I'm undecided if it's worth it 🤷‍♂️